crates/biome_css_analyze/src/assist/source/use_sorted_properties.rs

```diff
@@ -32,32 +32,20 @@ declare_source_rule! {
     ///
     /// The ordering is roughly:
     /// 1. Custom properties
-    /// 1. Layout properties (display, flex, grid)
-    /// 1. Margin and padding properties
-    /// 1. Typography properties (font, color)
-    /// 1. Interaction properties (pointer-events, visibility)
-    /// 1. Background and border properties
-    /// 1. Transition and animation properties
-    /// 1. Nested rules, media queries and other at-rules
+    /// 2. Layout properties (display, flex, grid)
+    /// 3. Margin and padding properties
+    /// 4. Typography properties (font, color)
+    /// 5. Interaction properties (pointer-events, visibility)
+    /// 6. Background and border properties
+    /// 7. Transition and animation properties
+    /// 8. Nested rules, media queries and other at-rules
```

Comment:
The number `1.` was there on purpose. It's a small trick. If you need to add a new item to the list, you don't need to update all the others, and it's always rendered correctly

Reply:
sorry, change back